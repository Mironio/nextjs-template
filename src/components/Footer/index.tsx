export default function Footer() {
  return (
    <footer className="pb-6">
      Powered by Miron Kalenta
    </footer>
  );
}
