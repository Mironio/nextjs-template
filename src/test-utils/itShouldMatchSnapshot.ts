import { expect, it } from 'vitest';
import { JSX } from 'react';

export default function itShouldMatchSnapshot(jsx: JSX.Element) {
  it('should match snapshot', () => {
    expect(jsx).toMatchSnapshot();
  });
}
