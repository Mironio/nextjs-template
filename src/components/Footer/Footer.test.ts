import itShouldMatchSnapshot from '@/test-utils/itShouldMatchSnapshot';
import Footer from './index';

itShouldMatchSnapshot(Footer());
