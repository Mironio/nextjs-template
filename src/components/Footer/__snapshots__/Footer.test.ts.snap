// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
<footer
  className="pb-6"
>
  Powered by Miron Kalenta
</footer>
`;
